File: C/BEGINNER/1046 Game Time.c
#include <stdio.h>
 
int main() 
{
    int startHour, endHour;
    scanf("%d %d", &startHour, &endHour);
    if (startHour == endHour)
        printf("O JOGO DUROU 24 HORA(S)\n");
    else if (startHour > endHour)
        printf("O JOGO DUROU %d HORA(S)\n", (24 - (startHour - endHour)));
    else
        printf("O JOGO DUROU %d HORA(S)\n", (endHour - startHour));
    return 0;
}

File: Python/BEGINNER/1046 Game Time.py
start_hour, final_hour = map(int, input().split())
if start_hour > final_hour:
    result = ((final_hour + 24) - start_hour)
elif start_hour < final_hour:
    result = (final_hour - start_hour)
else:
    result = 24
print('O JOGO DUROU %d HORA(S)' % result)
